index.d.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, CSSProperties, SFC } from "react";
+
+export interface ScrollbarRendererProps {
+    className?: string | string[];
```

Comment:
It looks like this can be tightened to `className: string`, since `join` is always called before the inner renderers are invoked.

Reply:
But what if i want to pass an array of strings? It is quite popular practice, as far as i know, and i'm using it myself sometimes.  
Anyway i dont see any sense in tightening this one.